controllers/match_controller.py

```diff
@@ -16,6 +16,17 @@ class MatchDetail(CacheableHandler):
     CACHE_VERSION = 4
     CACHE_KEY_FORMAT = "match_detail_{}"  # (match_key)
 
+    defense_render_names = {
```

Comment:
This year specific stuff (along with `add_defense_render_names`) is awkward in the default match controller... I'm trying to think of a better way to do this.